### DISCUSSION TITLE
v2.5.1

### DISCUSSION THREAD

**Discussion by mashb1t** [lllyasviel/Fooocus#3360] [General]
## What's Changed
* docs: update download URL in readme
* fix: use type pil for image upload to prevent conversion to png through temp file by @mashb1t in https://github.com/mashb1t/Fooocus/pull/58
* fix: allow reading of metadata from jpeg, jpg and webp again by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3301
* fix: correctly debug preprocessor again by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3332
* docs: update attributes and add add inline prompt features section to readme by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3333
* feat: add checkbox, config and handling for saving only the final enhanced image by @mashb1t in https://github.com/mashb1t/Fooocus/pull/61
* feat: sort enhance images by @mashb1t in https://github.com/mashb1t/Fooocus/pull/62

**Full Changelog**: https://github.com/lllyasviel/Fooocus/compare/v2.5.0...v2.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/lllyasviel/Fooocus/releases/tag/v2.5.1'>v2.5.1</a>.</em>

**Comment by mashb1t** [comment]
Excerpt of changelog:
https://github.com/lllyasviel/Fooocus/blob/c4ce2ce600792213c2e1a7b4f923f177a3e1ef13/update_log.md?plain=1#L1-L9

**Reply by Deij123** [reply]
как обновить фокус?

**Reply by Deij123** [reply]
2,5,0 есть для скачки а дальше как?
